### PR TITLE
Added My info

### PR DESCRIPTION
**Pull request by Sharjeeyl**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ contributions.forEach((contribution) => {
   // Populate the card with contribution data
   const image = document.createElement("img");
   image.src = contribution.image;
-  image.alt = "Student Image";
+  image.alt = "./image/sharjeel.jpeg";
```
Comment (mfaeezshabbir) [inline script.js:20]: Don't Do anything else kindly

**Review by mfaeezshabbir — CHANGES_REQUESTED** [review]
Kindly don't replace anything with your code just copy those four lines and add you info in script.js file and then push again

**Comment by Yash-Ainapure** [comment]
is this issue still open?I wanted to contribute some

**Comment by mfaeezshabbir** [comment]
> is this issue still open?I wanted to contribute some

Yes

**Pull request closed without merging**